app.js

```diff
@@ -0,0 +1,132 @@
+/* Esta funcion simula el test de tipo de piel de la pagina. En lugar de usar checkboxes, el usuario responde preguntas (si/no). Se mantiene la logica del test original, pero adaptada a entrada/salida por promt y alert */
+
+// funcion para el test de tipo de piel
+function iniciarTest() {
+    // lista de preguntas con su tipo asociado
+    const preguntas = [
```

Comment:
¿Estas preguntas ya van a estar precargadas?¿Por qué no utilizar una clase Test que las contenga? 
```suggestion
class Test {
  constructor() {
    this.preguntas = [];
  }
 ```

Reply:
Pensar en una clase pregunta
```suggestion
class Pregunta {
    constructor(texto, tipo) {
        this.texto = texto;
        this.tipo = tipo;
    }
}```

Reply:
```suggestion
class Test {
    constructor() {
        this.preguntas = [
            new Pregunta("Tu piel se siente tirante después de lavarla?", "seca"),
            new Pregunta("Tu piel se siente aceitosa al tacto?", "grasa"),
            new Pregunta("Tienes zonas grasas (nariz, frente) y otras secas (mejillas)?", "mixta"),
            new Pregunta("Tu piel brilla pocas horas después de lavarla?", "grasa"),
            new Pregunta("Tu piel se ve opaca o apagada?", "seca"),
            new Pregunta("Te tienes que aplicar crema humectante varias veces al día?", "seca"),
            new Pregunta("Tu piel se siente cómoda sin resequedad ni grasitud?", "normal"),
            new Pregunta("Tu piel se enrojece fácilmente o reacciona mal a productos?", "sensible")
        ];
   ...     
  }      ```